proposals/NNNN-audio-source-am-fm-xm.md

```diff
@@ -0,0 +1,57 @@
+# Audio Source AM/FM/XM
+
+* Proposal: [SDL-NNNN](NNNN-audio-source-am-fm-xm.md)
+* Author: [Zhimin Yang](https://github.com/smartdevicelink/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [iOS / Android / RPC / Core / Web Hmi ]
+
+## Introduction
+
+SDL remote control allows a mobile application to change the current audio source. When an application sets the audio source to `RADIO_TUNER`, the vehicle is supposed to use the last known/used radio band (AM/FM/XM) and frequency/station of the tuner. However, the application has no knowledge of last used radio band before sending such a request. The vehicle may or may not store the last used radio band. Therefore, the result of setting the audio source to `RADIO_TUNER` is unknown to the application. It is better for an application to set the audio source directly to `AM`, `FM` or `XM` (XM is for SiriusXM).
+
+## Motivation
+To give applications direct control of which radio band or SiriusXM radio they want to set as the audio source, we split `RADIO_TUNER` with detailed options.
+
+
+## Proposed solution
+
+In this proposal, we propose to replace the `RADIO_TUNER` with three enumeration elements `AM`, `FM` and `XM`.
+We have the same changes for both the mobile_api and hmi_api.
+
+```xml
+  <enum name="PrimaryAudioSource">
+    <description>Reflects the current primary audio source (if selected).</description>
+    ...
+    <element name="MOBILE_APP">
+    </element>
+-    <element name="RADIO_TUNER">
+-      <description>Radio may be on AM, FM or XM</description>
+-    </element>
++    <element name="AM">
++    </element>
++    <element name="FM">
++    </element>
++    <element name="XM">
++    </element>
++    <element name="DAB">
++    </element>
+  </enum>
+```
+
+Because the parameters (audio source, radio band, and radio frequency) belong to two different remote control moduleTypes, in order to set the audio source to a specific radio station or radio frequency, a mobile application needs to send two `setInteriorVehicleData` requests in sequence. The firsts request sets the audio source to `AM`/`FM`/`XM` with targeted `moduleType=AUDIO`. The second request sets the desired frequency of `AM`/`FM` radio or station number of `XM` radio with targeted `moduleType=RADIO`. This is true regardless of this proposal.
```

Comment:
Please update to "The first request sets..."

Reply:
updated